.travis.yml

```diff
@@ -73,7 +73,7 @@ script:
   - python manage.py check_for_endpoint_documentation
   - dredd > dredd-results.txt && echo '! grep -E "^[warn:|error:]" dredd-results.txt' | bash
   - pytest --durations 50 --ignore-glob='**/tests/integration/*' --cov=usaspending_api --cov-report= --reuse-db -rsx
-  - pytest --durations 50 --override-ini=python_files='**/tests/integration/test_*.py **/tests/integration/*_test.py' --cov=usaspending_api --cov-append --cov-report term --cov-report xml:coverage.xml --reuse-db -rsx
```

Comment:
is the `*` even nessicary? In my experience pytest lets you just run a folder and will deal with directory traversal by itself.

Reply:
I don't think it adds or hurts anything but added to make sure we test the inverse of the pytest run above.